staging/src/k8s.io/csi-api/pkg/apis/csi/v1alpha1/types.go

```diff
@@ -18,6 +18,15 @@ package v1alpha1
 
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+type PluginMechanism string
```

Comment:
Sorry, I changed my mind halfway through writing my last set of comments, I should have asked for "VolumePluginMechanism"

Reply:
now the type and the field are the same, not sure if that's an issue or not: `VolumePluginMechanism VolumePluginMechanism `json:"volumePluginMechanism"``